server/routes/feedback.js

```diff
@@ -0,0 +1,18 @@
+const feedbackController = require('../controllers//feedbackController');
+const passport = require('passport');
+const express = require('express');
+const router = express.Router();
+
+router.post(
+  '/',
+  passport.authenticate('jwt', { session: false }),
+  feedbackController.createFeedback
+);
```

Comment:
I was thinking about these specific routes, and realized the routes should probably be in an interviews router and the routes should look like:

PUT /interviews/:interviewId/feedback
- This would allow you to edit the feedback of that specific interview - the body would be the rest of the feedback data - then the current user will always review the other user in that interview object

GET /interviews/:interviewId
- this should get the whole interview object with the feedbacks